python/mxnet/ndarray/numpy/_op.py

```diff
@@ -28,8 +28,120 @@
 __all__ = ['zeros', 'ones', 'maximum', 'minimum', 'stack', 'arange', 'argmax',
            'add', 'subtract', 'multiply', 'divide', 'mod', 'power', 'concatenate',
            'clip', 'split', 'swapaxes', 'expand_dims', 'tile', 'linspace',
-           'sin', 'cos', 'sinh', 'cosh', 'log10', 'sqrt']
+           'sin', 'cos', 'sinh', 'cosh', 'log10', 'sqrt', 'absolute', 'cbrt', 
+           'arccos']
 
+@set_module('mxnet.ndarray.numpy')
+def absolute(x, out=None, **kwargs):
+    r"""
+    Calculate the absolute value element-wise.
+    np.abs is a shorthand for this function.
+
+    Parameters
+    ----------
+    x : ndarray
+    Input array.
+
+    out : ndarray, optional
```

Comment:
Please check whether ``out`` supports tuple as input.

Reply:
Thx and it doesn't.